eth-types/src/evm_types/memory.rs

```diff
@@ -285,6 +319,18 @@ impl Memory {
     pub fn word_size(&self) -> usize {
         self.0.len() / 32
     }
+
+    /// Resize the memory for at least length and align to 32 bytes.
+    pub fn extend_at_least(&mut self, minimal_length: usize) {
```

Comment:
```suggestion
    pub fn expand(&mut self, length: usize) {
```

Reply:
the old name is telling caller that this function might not expand the memory to given length. the length is aligned to 32 bytes.